src/Lecture2.hs

```diff
@@ -158,13 +182,62 @@ You're free to define any helper functions.
 -}
 
 -- some help in the beginning ;)
+type Health = Int
+type Endurance = Int
+type Power = Int
```

Comment:
This is a nice start! But generally these types would be more useful if they will be newtypes 🙂